applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -236,12 +236,19 @@ int sw_codec_decode(uint8_t const *const encoded_data, size_t encoded_size, bool
 
 		switch (m_config.decoder.channel_mode) {
 		case SW_CODEC_MONO: {
-			ret = sw_codec_lc3_dec_run(encoded_data, encoded_size,
-						   LC3_PCM_NUM_BYTES_MONO, 0, pcm_data_mono,
-						   (uint16_t *)&pcm_size_session, bad_frame);
-			if (ret) {
-				return ret;
+
+			if (bad_frame && IS_ENABLED(CONFIG_SW_CODEC_OVERRIDE_PLC)) {
+				memset(pcm_data_mono, 0, PCM_NUM_BYTES_MONO);
+				pcm_size_session = PCM_NUM_BYTES_MONO; // num bytes written
```

Comment:
The // comment is a C++ style comment, so either change it to /* */ or remove it. Personally I don't we need the comment there

Reply:
The checkpatch script should throw errors on "//" comments. It can be used in the pre-commit Git hook.

Reply:
![image](https://user-images.githubusercontent.com/88771755/168017240-30db9cd2-c7fb-41bf-8f15-536a61380c61.png)

No error message indeed.

Reply:
Removed comment.